group_vars/postgresql/production.yml

```diff
@@ -2,20 +2,26 @@
 postgresql_version: "13"
 db_clusteradmin_user: "postgres"
 db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
+replication_user_password: "{{ vault_replicant_password }}"
 postgresql_settings:
   - name: port
     value: 5432
   - name: listen_addresses
     value: "*"
 postgresql_ext_install_postgis: true
 postgresql_databases:
-  - name: test_db
+  - name: replicant
 postgresql_users:
   - name: postgres
     password: "{{ vault_postgres_admin_password }}"
     db: "postgres"
     priv: "ALL"
     role_attr_flags: "SUPERUSER"
+  - name: replicant
+    password: "{{ vault_postgres_admin_password }}"
```

Comment:
Should this be `"{{ vault_replicant_password }}"` or `"{{ replication_user_password }}"`? When we looked at the staging server, the correct database pw was in use, so this may take some more investigation.